utils/unittest/fixtures.go

```diff
@@ -105,6 +105,24 @@ func BlockFixture() flow.Block {
 	return BlockWithParentFixture(&header)
 }
 
+func FullBlockFixture() flow.Block {
```

Comment:
🤔 I am confused what the difference is between `FullBlockFixture` and `BlockFixture`.
It seems that `BlockFixture` (which uses `PayloadFixture`) only populates `Guarantees` and `Seals` (which seems kind of arbitrary). Here you re-generate the  `Seals` (why?) and add `Receipts`.

Is yet an additional fixture really needed?

#### Suggestion:
extend `PayloadFixture` 
https://github.com/onflow/flow-go/blob/2375004eb5ecee3a16889403b1b75eea0832ee2b/utils/unittest/fixtures.go#L167-L176 to also include  `Receipts`.

Reply:
I think we still need the `FullBlockFixture`, because some tests need blocks that doesn't have seals or receipts, otherwise they break.

And we can't use `FullBlockFixture` in those test cases, because the random seals and receipts we generated are invalid, so we have different scenarios, better just separate them out.